### ISSUE TITLE
Pre-work Checklist: Developer: Yang "Ino" Popper

### ISSUE THREAD

**Issue by ino-iosdev** [hackforla/website#6885]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Self Assign this issue (gear in right side panel)
- [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
- [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ino-iosdev** [comment]
i. Pretty flexible with time recently.
ii. The next 2-3 days hopefully.

**Comment by ino-iosdev** [comment]
### Availability for this week:

Pretty available for the most part.

### My estimated ETA for completing this issue:

Initial ETA is 2-3 days.

### How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

It took me 5-6 hours total to finish the pre-work of this issue, I'm ready to find a good first issue after this one, I already attended my first meeting tonight on 5/23 7pm

### Provide Update

1. Progress - completed within estimated time
2. Blockers - lots of new stuff I have to learn still, so extensive reading and asking around for help took time
3. Availability - pretty flexible and available
4. ETA - currently completed

**Comment by HackforLABot** [comment]
Hello @ino-iosdev, we appreciate you taking on issue #7645, however it looks like you've already worked on enough issues of this complexity. Try checking out some of the issues of the next complexity from the Prioritized Backlog :)

We are going to unassign you from this issue so you can take on another issue.

Hfla appreciates you! :)

P.S. There is one exception to this rule/automation, and that is if you were away for a long time, and need to do the issue ladder again. If that is the case, please post the following note on the issue and on your Skills Issue (Pre-work Checklist). A Merge team member will reassign you to this issue, and will help you get assigned to subsequent issues up to medium size.

```
I am returning after a significant time away, and need to do the issue ladder again. Please assign me back to this issue.
```

**Comment by ino-iosdev** [comment]
I am returning after a significant time away, and need to do the issue ladder again. Please assign me back to this issue.

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the 'To Update !' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, January 20, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
Hello @ino-iosdev, we appreciate you taking on issue #7481, however it looks like you've already worked on enough issues of this complexity. Try checking out some of the issues of the next complexity from the Prioritized Backlog :)

We are going to unassign you from this issue so you can take on another issue.

Hfla appreciates you! :)

P.S. There is one exception to this rule/automation, and that is if you were away for a long time, and need to do the issue ladder again. If that is the case, please post the following note on the issue and on your Skills Issue (Pre-work Checklist). A Merge team member will reassign you to this issue, and will help you get assigned to subsequent issues up to medium size.

```
I am returning after a significant time away, and need to do the issue ladder again. Please assign me back to this issue.
```

**Comment by HackforLABot** [comment]
Hello @ino-iosdev, we appreciate you taking on issue #7448, however it looks like you've already worked on enough issues of this complexity. Try checking out some of the issues of the next complexity from the Prioritized Backlog :)

We are going to unassign you from this issue so you can take on another issue.

Hfla appreciates you! :)

P.S. There is one exception to this rule/automation, and that is if you were away for a long time, and need to do the issue ladder again. If that is the case, please post the following note on the issue and on your Skills Issue (Pre-work Checklist). A Merge team member will reassign you to this issue, and will help you get assigned to subsequent issues up to medium size.

```
I am returning after a significant time away, and need to do the issue ladder again. Please assign me back to this issue.
```

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, February 3, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, February 10, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, February 17, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, February 24, 2025 at 11:05 PM PST.</sub>

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, March 3, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, March 11, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, March 18, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@ino-iosdev

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, March 25, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
## Activity Log: ino-iosdev
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- ino-iosdev opened: https://github.com/hackforla/website/issues/6885 at 05/22/2024, 12:16 AM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6885 at 05/22/2024, 12:16 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6885#issuecomment-2124047641 at 05/22/2024, 12:19 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6885#issuecomment-2128872186 at 05/24/2024, 01:10 AM PDT
- ino-iosdev closed as completed: https://github.com/hackforla/website/issues/6885#event-12920899109 at 05/24/2024, 01:10 AM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6799 at 05/24/2024, 02:14 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6799#issuecomment-2129058074 at 05/24/2024, 02:25 AM PDT
- ino-iosdev opened: https://github.com/hackforla/website/pull/6894 at 05/24/2024, 04:22 PM PDT
- ino-iosdev PR was merged: https://github.com/hackforla/website/pull/6894#event-12937456279 at 05/26/2024, 09:54 AM PDT
- ino-iosdev issue closed by PR 6894: https://github.com/hackforla/website/issues/6799#event-12937456323 at 05/26/2024, 09:54 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6901#issuecomment-2138047768 at 05/29/2024, 11:44 AM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6901#pullrequestreview-2086537863 at 05/29/2024, 02:21 PM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6864 at 05/29/2024, 02:44 PM PDT
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/6864 at 05/29/2024, 02:44 PM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6798 at 05/29/2024, 02:48 PM PDT
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/6798 at 05/29/2024, 02:48 PM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6730 at 05/29/2024, 03:15 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6730#issuecomment-2138350080 at 05/29/2024, 03:17 PM PDT
- ino-iosdev opened: https://github.com/hackforla/website/pull/6914 at 05/29/2024, 08:06 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6730#issuecomment-2138594694 at 05/29/2024, 08:12 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6913#issuecomment-2138602873 at 05/29/2024, 08:22 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6913#issuecomment-2138610921 at 05/29/2024, 08:32 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6913#pullrequestreview-2087179485 at 05/29/2024, 09:01 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6912#issuecomment-2138638892 at 05/29/2024, 09:09 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6912#pullrequestreview-2087201456 at 05/29/2024, 09:29 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6913#pullrequestreview-2089239981 at 05/30/2024, 12:44 PM PDT
- ino-iosdev closed: https://github.com/hackforla/website/pull/6914#event-12993507729 at 05/30/2024, 06:37 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6914#issuecomment-2141095619 at 05/30/2024, 06:44 PM PDT
- ino-iosdev opened: https://github.com/hackforla/website/pull/6917 at 05/30/2024, 11:13 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6916#issuecomment-2141321251 at 05/30/2024, 11:33 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6921#issuecomment-2143648107 at 06/01/2024, 05:41 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6921#issuecomment-2143648841 at 06/01/2024, 05:45 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6901#issuecomment-2143692700 at 06/01/2024, 09:28 PM PDT
- ino-iosdev PR was merged: https://github.com/hackforla/website/pull/6917#event-13012612378 at 06/02/2024, 10:44 AM PDT
- ino-iosdev issue closed by PR 6917: https://github.com/hackforla/website/issues/6730#event-13012612410 at 06/02/2024, 10:44 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6924#issuecomment-2144649441 at 06/03/2024, 01:53 AM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6924#pullrequestreview-2098358313 at 06/05/2024, 01:10 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6924#issuecomment-2149178776 at 06/05/2024, 01:18 AM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6941 at 06/05/2024, 01:29 AM PDT
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/6941 at 06/05/2024, 01:29 AM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6935 at 06/05/2024, 01:32 AM PDT
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/6935 at 06/05/2024, 01:32 AM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6908 at 06/05/2024, 01:35 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6908#issuecomment-2149214979 at 06/05/2024, 01:36 AM PDT
- ino-iosdev opened: https://github.com/hackforla/website/pull/6963 at 06/05/2024, 02:12 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6958#issuecomment-2150991903 at 06/05/2024, 02:27 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6958#pullrequestreview-2100340477 at 06/05/2024, 02:43 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6958#pullrequestreview-2100341459 at 06/05/2024, 02:44 PM PDT
- ino-iosdev PR was merged: https://github.com/hackforla/website/pull/6963#event-13069659611 at 06/06/2024, 09:51 AM PDT
- ino-iosdev issue closed by PR 6963: https://github.com/hackforla/website/issues/6908#event-13069659777 at 06/06/2024, 09:51 AM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6916#pullrequestreview-2103580482 at 06/06/2024, 08:47 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6916#pullrequestreview-2111706901 at 06/11/2024, 06:10 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6982#issuecomment-2161879846 at 06/11/2024, 06:17 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6982#pullrequestreview-2111756485 at 06/11/2024, 06:51 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6984#pullrequestreview-2111805745 at 06/11/2024, 07:15 PM PDT
- ino-iosdev opened: https://github.com/hackforla/website/issues/6990 at 06/11/2024, 08:07 PM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6755 at 06/11/2024, 08:43 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6755#issuecomment-2162048275 at 06/11/2024, 08:46 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6982#pullrequestreview-2111895852 at 06/11/2024, 08:52 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6989#pullrequestreview-2111939502 at 06/11/2024, 09:44 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6980#issuecomment-2163455284 at 06/12/2024, 09:25 AM PDT
- ino-iosdev issue closed as completed: https://github.com/hackforla/website/issues/6755#event-13134610280 at 06/12/2024, 09:34 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6980#issuecomment-2163473186 at 06/12/2024, 09:36 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6980#issuecomment-2163493512 at 06/12/2024, 09:48 AM PDT
- ino-iosdev merged: https://github.com/hackforla/website/pull/6980#event-13134761820 at 06/12/2024, 09:48 AM PDT
- ino-iosdev reopened: https://github.com/hackforla/website/pull/6980#event-13134761820 at 06/12/2024, 09:53 AM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/pull/6984#issuecomment-2163543607 at 06/12/2024, 10:16 AM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6980#pullrequestreview-2114210819 at 06/12/2024, 02:00 PM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6850 at 06/12/2024, 02:45 PM PDT
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6850#issuecomment-2163956515 at 06/12/2024, 02:46 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6980#pullrequestreview-2114288571 at 06/12/2024, 02:50 PM PDT
- ino-iosdev opened: https://github.com/hackforla/website/pull/6999 at 06/12/2024, 05:04 PM PDT
- ino-iosdev PR was merged: https://github.com/hackforla/website/pull/6999#event-13166269856 at 06/14/2024, 04:19 PM PDT
- ino-iosdev issue closed by PR 6999: https://github.com/hackforla/website/issues/6850#event-13166269937 at 06/14/2024, 04:19 PM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6982#pullrequestreview-2123212855 at 06/17/2024, 08:24 AM PDT
- ino-iosdev submitted review: https://github.com/hackforla/website/pull/6984#pullrequestreview-2136824306 at 06/24/2024, 02:33 PM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/6990 at 10/06/2024, 10:49 AM PDT
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/6990 at 10/06/2024, 02:41 PM PDT
- ino-iosdev assigned: https://github.com/hackforla/website/issues/7645 at 01/16/2025, 06:53 PM PST
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/7645 at 01/16/2025, 06:53 PM PST
- ino-iosdev commented on: https://github.com/hackforla/website/issues/7645#issuecomment-2597330105 at 01/16/2025, 06:57 PM PST
- ino-iosdev commented on: https://github.com/hackforla/website/issues/6885#issuecomment-2597330910 at 01/16/2025, 06:57 PM PST
- ino-iosdev assigned: https://github.com/hackforla/website/issues/7645 at 01/20/2025, 06:32 PM PST
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/7645 at 01/20/2025, 06:33 PM PST
- ino-iosdev assigned: https://github.com/hackforla/website/issues/7645 at 01/20/2025, 06:34 PM PST
- ino-iosdev commented on: https://github.com/hackforla/website/issues/7645#issuecomment-2603794773 at 01/20/2025, 10:50 PM PST
- ino-iosdev opened: https://github.com/hackforla/website/pull/7851 at 01/24/2025, 11:57 PM PST
- ino-iosdev commented on: https://github.com/hackforla/website/pull/7851#issuecomment-2613833692 at 01/25/2025, 12:04 AM PST
- ino-iosdev closed as completed: https://github.com/hackforla/website/issues/6885#event-16066803811 at 01/25/2025, 12:07 AM PST
- ino-iosdev commented on: https://github.com/hackforla/website/pull/7851#issuecomment-2628785724 at 01/31/2025, 09:28 PM PST
- ino-iosdev PR was merged: https://github.com/hackforla/website/pull/7851#event-16153903224 at 02/02/2025, 02:29 PM PST
- ino-iosdev issue closed by PR 7851: https://github.com/hackforla/website/issues/7645#event-16153903272 at 02/02/2025, 02:29 PM PST
- ino-iosdev assigned: https://github.com/hackforla/website/issues/7481 at 02/02/2025, 08:09 PM PST
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/7481 at 02/02/2025, 08:09 PM PST
- ino-iosdev assigned: https://github.com/hackforla/website/issues/7448 at 02/02/2025, 08:16 PM PST
- ino-iosdev unassigned: https://github.com/hackforla/website/issues/7448 at 02/02/2025, 08:16 PM PST
- ino-iosdev closed as completed: https://github.com/hackforla/website/issues/6885#event-17070035258 at 04/01/2025, 12:36 AM PDT
